prov/efa/src/rdm/rxr_pkt_entry.c

```diff
@@ -428,15 +425,15 @@ ssize_t rxr_pkt_entry_send(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry,
 	send_wr->wr.ud.remote_qpn = conn->ep_addr->qpn;
 	send_wr->wr.ud.remote_qkey = conn->ep_addr->qkey;
 
-	efa_ep->base_ep.xmit_more_wr_tail->next = send_wr;
-	efa_ep->base_ep.xmit_more_wr_tail = send_wr;
+	ep->base_ep.xmit_more_wr_tail->next = send_wr;
+	ep->base_ep.xmit_more_wr_tail = send_wr;
 
 	if (flags & FI_MORE) {
 		rxr_ep_record_tx_op_submitted(ep, pkt_entry);
 		return 0;
 	}
 
-	ret = efa_post_flush(efa_ep, &bad_wr, false /* don't free ibv_send_wr */);
+	ret = efa_rdm_post_flush(ep, &bad_wr);
```

Comment:
better name the function `efa_rdm_ep_post_flush`

Reply:
done